wrangler-service/src/main/java/io/cdap/wrangler/service/directive/DirectivesHandler.java

```diff
@@ -1123,6 +1130,17 @@ private static Row createUberRecord(List<Row> rows) {
         } catch (RecipeException e) {
           throw new BadRequestException(e.getMessage(), e);
         }
+
+        List<ErrorRecordBase> errors = executor.errors()
+            .stream()
+            .filter(ErrorRecord::isShownInWrangler)
+            // Use slim error record to reduce message size.
+            .map(SlimErrorRecord::new)
```

Comment:
is the eventual response eventually serialized as a SlimErrorRecord? I would think it gets serialized as an ErrorRecordBase since that's what's returned by ErrorRecordsException.getErrorRecords()?

Reply:
It gets serialized as SlimErrorRecord, since that's the actual type.